### DISCUSSION TITLE
Does marking a message as spam in the Webmail/Quarantine train the spam engine?

### DISCUSSION THREAD

**Discussion by binarydad** [modoboa/modoboa#2473] [General]
Do any of these options train the spam engine to fine-tune spam vs ham? Thank you.

![image](https://user-images.githubusercontent.com/1760607/156836220-1ff1439a-0942-4a0f-bf60-401cac30cd38.png)

![image](https://user-images.githubusercontent.com/1760607/156836301-767cd831-8b74-4fc2-9b2c-d4161cfa54c5.png)

**Comment by ionescu77** [comment]
Hi, there is not much activity sadly. I have turned-off spam filtering on my Digital Ocean droplet as 2G RAM was not enough.

I will be interested also to know about the spam filtering, as I am getting a lot - even simple spam (like emails with no valid domain name or with invalid DKIM signature).

I'll try to make a local installation and give it a try when I have time.

**Comment by ionescu77** [comment]
I see you have found this other thread.

I guess training is disabled by default

https://github.com/modoboa/modoboa/discussions/2445
